### PR TITLE
Polaris sync 20170630

### PR DESCRIPTION
**Pull request by Leshana**
### Merged Changes

Eliminates Construction borgs by merging them into Engineering borgs - https://github.com/PolarisSS13/Polaris/pull/3540
Adds AOOC functionality for antags - https://github.com/PolarisSS13/Polaris/pull/3543
Nerfs the siemens on most hardsuits - https://github.com/PolarisSS13/Polaris/pull/3523
Adds snowsuits - https://github.com/PolarisSS13/Polaris/pull/3513
Ports Bay/Paradise Defibs - https://github.com/PolarisSS13/Polaris/pull/3528
Removes staff exemption for AFK kicking - https://github.com/PolarisSS13/Polaris/pull/3544
Adds a new Infiltrator Gamemode - https://github.com/PolarisSS13/Polaris/pull/3534
Updates surgical tools and brains - https://github.com/PolarisSS13/Polaris/pull/3536
Replaces baguette sprite - https://github.com/PolarisSS13/Polaris/pull/3535
Ports inflatable dispensers from Bay - https://github.com/PolarisSS13/Polaris/pull/3510
Adds ling claw power - https://github.com/PolarisSS13/Polaris/pull/3448
Makes the CEs hardsuit and Engineering hardsuit temperature resistant - https://github.com/PolarisSS13/Polaris/pull/3514

### Merged Fixes
Nertii did Aro's fix for Say/LOOC - https://github.com/PolarisSS13/Polaris/pull/3545
Removes a comma that allowed nuka_cola to spawn in flasks - https://github.com/PolarisSS13/Polaris/pull/3539
Blast Door repair & damage fix - https://github.com/PolarisSS13/Polaris/pull/3538

### Post-Merge Changes
* Fixes ammo paths changed by PolarisSS13/Polaris#3541 in VOREStation custom code.
* Updates VOREStation tether map to use the new defibs from PolarisSS13/Polaris#3528

### CODE REVIEW

**Comment by SpadesNeil** [comment]
I did a thing.

File: code/_helpers/game.dm [hunk 1]
```diff
@@ -281,15 +281,15 @@
 
 	//A list of every mob with a client
 	for(var/mob in player_list)
-		if(isnull(mob)) //VOREStation Add - Fix for a weird bug with nulls in this list
-			player_list -= mob //VOREStation Add
-			continue //VOREStation Add
+		if(!istype(mob, /mob))
+			crash_with("There is a null or non-mob reference inside player_list.")
```
Comment (SpadesNeil) [inline code/_helpers/game.dm]: This is removed on purpose?

Comment (Leshana) [inline code/_helpers/game.dm]: Yes, replaced with the version Polaris put in.  Altho on a second look each had advantages. I think I will revise to be best of both.

**Comment by SpadesNeil** [comment]
Gonna merge this for now.

**Pull request merged**